charactersheet/charactersheet/templates/dm/encounter_sections/player_text_section.tmpl.html

```diff
@@ -14,6 +14,7 @@
         Description
         <span data-bind="css: sortArrow('description')"></span>
       </th>
+      <th></th>
```

Comment:
Should the whole column be disabled when `shouldShowPushButton` is false? Currently, the column exists, but is empty when a user is not connected.